localstack/services/events/provider.py

```diff
@@ -143,6 +156,21 @@ def format_event(event: PutEventsRequestEntry, region: str, account_id: str) ->
     return formatted_event
 
 
+def get_bus_or_rule(arn: Arn) -> ResourceType:
```

Comment:
maybe `get_resource_type()`

Reply:
refactored in: https://github.com/localstack/localstack/pull/10840/commits/ab40f18492de93569b5c43202f5aac1cccfd089c